### ISSUE TITLE
[BUG] New `SeigniorageRecipientsSnapshot` needs to be handled by the sidecar

### ISSUE THREAD

**Issue by devendran-m** [casper-network/casper-sidecar#352]
Ticket to track and resolve the issue to handle new `SeigniorageRecipientsSnapshot` by the sidecar.

**Comment by devendran-m** [comment]
This was blocking AJ's tests. Jakub has unblocked AJ with a quick and dirty fix. Waiting on Ed's changes for the permanent fix.

**Comment by devendran-m** [comment]
This is superseded by #388, therefore closing.
